common/version.go

```diff
@@ -5,7 +5,7 @@ import "github.com/nspcc-dev/neo-go/pkg/interop/native/std"
 const (
 	major = 0
 	minor = 17
-	patch = 0
+	patch = 5
```

Comment:
Don't we need to update correspondingly the `prevMinor` and `prevPatch` as far? I'm not so deep in this part of contract updates, so maybe we don't need to.

Reply:
btw, why not `1`?

Reply:
`Prev` constants are OK as is, it's the oldest version we support upgrading from and so far we have not removed any of the migration code. The value is somewhere in the middle of 17.0 and 18.0. 1 makes sense in general, but it doesn't matter much here since there are new APIs.